README.md

```diff
@@ -16,4 +16,24 @@ You open up the command palette and run the **Sync Letterboxd Diary** command. I
 - Gave [3 stars to Lessons in Chemistry](https://letterboxd.com/fleker/film/lessons-in-chemistry/) on [[2024-03-19]]
 ```
 
-This plugin could use your feedback and help to make it a success!
\ No newline at end of file
+This plugin could use your feedback and help to make it a success!
+
+## Usage:
+
+### Add Last Watched Movie
+
+This command will add the last watched movie in the present active editor. This will be done in a special block. For example:
+
+```md
+>[!Last Movie Logged]+ 
+> [The Tragedy of Macbeth, 2021 - ★★★★](https://letterboxd.com/ati_zaf/film/the-tragedy-of-macbeth/) on 2024-08-16 
+>  <p><img src="https://a.ltrbxd.com/resized/film-poster/5/1/9/0/5/2/519052-the-tragedy-of-macbeth-0-600-0-900-crop.jpg?v=2feba2c908"/></p> <p>Visually stunning. The use of light and shadow is remarkable in this movie. Denzel's monologue about the dagger was incredible.<br />Yet i felt something was amiss. Maybe it was acting by the rest of the cast. Can't put a pin on it.</p>   
+```
+
+### Add Last 2 Watched Movies
```

Comment:
No, that was always a temporary addition for my own use case. This will be removed. Last movie watched, movies watched that day and all movies would be enough. I will get to this sometime this week, been busy with other projects.

Reply:
Okay looking forward to it. Take your time.